### PR TITLE
gfortran 10 support

### PR DESCRIPTION
**Pull request by akrherz**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
*  Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

@conda-forge-admin, please rerender

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -65,7 +65,6 @@ about:
 extra:
   recipe-maintainers:
     - eugeneburger
-    - josborne-noaa
```
Comment (ocefpaf) [inline recipe/meta.yaml:68]: Did you communicate this with @josborne-noaa?

Comment (akrherz) [reply recipe/meta.yaml:68]: The linter complains otherwise and says this user does not exist.

Comment (ocefpaf) [reply recipe/meta.yaml:68]: Oh! I guess he left NOAA. @eugeneburger if there is a new PyFerret developer we can add them here in the future.

**Comment by akrherz** [comment]
looks like `-fallow-argument-mismatch` is needed too.

**Comment by akrherz** [comment]
Sweet, hoping to go green here :)  @ocefpaf you happy with this?

**Pull request merged**